examples/content_management_examples/server/examples/todos/todos.ts

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import {
+  ContentStorage,
+  StorageContext,
+  ContentManagementServerSetup,
+} from '@kbn/content-management-plugin/server';
+import { v4 } from 'uuid';
+import {
+  createInSchema,
+  searchInSchema,
+  Todo,
+  TODO_CONTENT_ID,
+  updateInSchema,
+  TodoSearchOut,
+  TodoCreateOut,
+  TodoUpdateOut,
+  TodoDeleteOut,
+  TodoGetOut,
+  createOutSchema,
+  getOutSchema,
+  updateOutSchema,
+  searchOutSchema,
+} from '../../../common/examples/todos';
+
+export const registerTodoContentType = ({
+  contentManagement,
+}: {
+  contentManagement: ContentManagementServerSetup;
+}) => {
+  contentManagement.register({
+    id: TODO_CONTENT_ID,
+    schemas: {
+      content: {
+        create: {
+          in: {
+            data: createInSchema,
+          },
+          out: {
+            result: createOutSchema,
+          },
+        },
+        update: {
+          in: {
+            data: updateInSchema,
+          },
+          out: {
+            result: updateOutSchema,
+          },
+        },
+        search: {
+          in: {
+            query: searchInSchema,
+          },
+          out: {
+            result: searchOutSchema,
+          },
+        },
+        get: {
+          // TODO: why no get in ?
+          out: {
+            result: getOutSchema,
+          },
+        },
+
+        // TODO: why no delete?
+      },
+    },
+    storage: new TodosStorage(),
+  });
+};
+
+class TodosStorage implements ContentStorage {
+  private db: Map<string, Todo> = new Map();
+
+  constructor() {
+    const id1 = v4();
+    this.db.set(id1, {
+      id: id1,
+      title: 'Learn Elasticsearch',
+      completed: true,
+    });
+    const id2 = v4();
+    this.db.set(id2, {
+      id: id2,
+      title: 'Learn Kibana',
+      completed: false,
+    });
+  }
+
+  // TODO: how to connect this to TodoGetIn?
+  // TODO: hot to connect this to input and output schema?
+  async get(ctx: StorageContext, id: string): Promise<TodoGetOut> {
+    return this.db.get(id)!;
+  }
+
+  async bulkGet(ctx: StorageContext, ids: string[]): Promise<TodoGetOut[]> {
+    return ids.map((id) => this.db.get(id)!);
+  }
+
+  async create(ctx: StorageContext, data: Omit<Todo, 'id' | 'completed'>): Promise<TodoCreateOut> {
```

Comment:
Same comment as above about the returned interface. We need to return _what the db returns_ not what the RPC client in the browser expects from calling our "get" procedure.